packages/api/tests/lib/test-cwSfExecutionEventUtils.js

```diff
@@ -0,0 +1,255 @@
+      getExecutionHistory: ({ executionArn }) => {
+        if (executionArn !== 'arn:aws:states:us-east-1:111122223333:execution:HelloWorld-StateMachine:my-execution-name') {
+          throw new Error(`Expected executionArn === 'arn:aws:states:us-east-1:111122223333:execution:HelloWorld-StateMachine:my-execution-name' but got ${executionArn}`);
+        }
+        return {
+          events: []
+        };
+      }
+    }
+  })(() => getFailedExecutionMessage(inputMessage));
+
+  t.deepEqual(actualResult, inputMessage);
+});
+
+test('getFailedExecutionMessage() returns the input message with the details from the last failed lambda step event in the exception field if the failed step exited event cannot be found', async (t) => {
+  const inputMessage = {
+    cumulus_meta: {
+      state_machine: 'arn:aws:states:us-east-1:111122223333:stateMachine:HelloWorld-StateMachine',
+      execution_name: 'my-execution-name'
+    }
+  };
+
+  const actualResult = await cwSfExecutionEventUtils.__with__({
+    StepFunctions: {
+      getExecutionHistory: ({ executionArn }) => {
+        if (executionArn !== 'arn:aws:states:us-east-1:111122223333:execution:HelloWorld-StateMachine:my-execution-name') {
+          throw new Error(`Expected executionArn === 'arn:aws:states:us-east-1:111122223333:execution:HelloWorld-StateMachine:my-execution-name' but got ${executionArn}`);
+        }
+        return {
+          events: [
+            {
+              // lastStepFailedEvent
+              type: 'LambdaFunctionFailed',
+              id: 1,
+              lambdaFunctionFailedEventDetails: {
+                type: 'really bad'
+              }
+            }
+          ]
+        };
+      }
+    }
+  })(() => getFailedExecutionMessage(inputMessage));
+
+  const expectedResult = {
+    ...inputMessage,
+    exception: {
+      type: 'really bad'
+    }
+  };
+
+  t.deepEqual(actualResult, expectedResult);
+});
+
+test('getFailedExecutionMessage() returns the input message with the details from the last failed activity step event in the exception field if the failed step exited event cannot be found', async (t) => {
+  const inputMessage = {
+    cumulus_meta: {
+      state_machine: 'arn:aws:states:us-east-1:111122223333:stateMachine:HelloWorld-StateMachine',
+      execution_name: 'my-execution-name'
+    }
+  };
+
+  const actualResult = await cwSfExecutionEventUtils.__with__({
+    StepFunctions: {
+      getExecutionHistory: ({ executionArn }) => {
+        if (executionArn !== 'arn:aws:states:us-east-1:111122223333:execution:HelloWorld-StateMachine:my-execution-name') {
+          throw new Error(`Expected executionArn === 'arn:aws:states:us-east-1:111122223333:execution:HelloWorld-StateMachine:my-execution-name' but got ${executionArn}`);
+        }
+        return {
+          events: [
+            {
+              // lastStepFailedEvent
+              type: 'ActivityFailed',
+              id: 1,
+              activityFailedEventDetails: 'my-details'
```

Comment:
This should be updated to be an object as in the previous test

Reply:
Fixed

Reply:
I'm not seeing the changes?

Reply:
Try now, I hadn't pushed yet.